build/pipelines/templates/helix-processtestresults-job.yml

```diff
@@ -7,7 +7,7 @@ parameters:
 
 jobs:
 - job: ProcessTestResults
-  condition: succeededOrFailed()
+  condition: ${{ parameters.condition }}
```

Comment:
Wait does it work if you just leave out the condition here? Is it because it is a *job*? I would have expected that condition was only overwritten because you overwrote it, like our cascading JSON stuff

Reply:
I'd rather just leave it here to match what the other Helix script does, if you don't mind.

Reply:
@kmahone tells me in Teams that templates don't have conditions or dependencies. Only jobs, stages, and tasks. So at this phase, we have to pass the conditions/dependencies down from the parent into the job as parameters and then tell the job to evaluate whatever it receives. So this wouldn't work how you think.